ASPXParser/ControlsData.cs

```diff
@@ -1,21 +1,94 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
 using System.Text;
+using System.Text.RegularExpressions;
+using HtmlAgilityPack;
 using Newtonsoft.Json;
 
 namespace ASPXParser
 {
     public class ControlsData
     {
-        IDictionary<String, ControlData> controlsNumbers = new Dictionary<String, ControlData>();
-        public ControlsData()
+        IDictionary<string, ControlData> controlsNumbers = new Dictionary<string, ControlData>();
+
+        public void GetData(IEnumerable<string> webFormsFiles)
```

Comment:
ControlsData => meant to be a entity and working on that entity... you can move the GetData logic to ControlParser if you don't like in Program file

Reply:
Good suggestion. I created separate parsers for ControlsData/ControlData and SolutionData. This separates the business logic from the data models.